doc/cephfs/shell.rst

```diff
@@ -0,0 +1,277 @@
+
```

Comment:
Please add:

`.. _cephfs-shell:`

at the top.

We will use that to reference to this document in the index.

Reply:
Then update `cephfs/index.rst` too in this PR.